libraries/chain/apply_context.cpp

```diff
@@ -1229,10 +1240,11 @@ void apply_context::add_ram_usage( account_name account, int64_t ram_delta, cons
    }
 }
 
-void apply_context::add_disk_usage( account_name account, int64_t disk_delta, const storage_usage_trace& trace ) {
-   trx_context.add_disk_usage( account, disk_delta, trace );
+void apply_context::add_disk_usage( account_name payer, int64_t disk_delta, const storage_usage_trace& trace ) {
```

Comment:
Same thing here, I'm not sure if this should be included.

@b1bart, thoughts on this?